### PR TITLE
Apply `black` to `astropy.tests`

### PR DESCRIPTION
**Pull request by WilliamJamieson**
### Description

Apply `black` to `astropy.tests` according to the process laid out by [APE 20](https://github.com/astropy/astropy-APEs/blob/main/APE20.rst).

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Do the proposed changes actually accomplish desired goals?
- [ ] Do the proposed changes follow the [Astropy coding guidelines](https://docs.astropy.org/en/latest/development/codeguide.html)?
- [ ] Are tests added/updated as required? If so, do they follow the [Astropy testing guidelines](https://docs.astropy.org/en/latest/development/testguide.html)?
- [ ] Are docs added/updated as required? If so, do they follow the [Astropy documentation guidelines](https://docs.astropy.org/en/latest/development/docguide.html#astropy-documentation-rules-and-guidelines)?
- [ ] Is rebase and/or squash necessary? If so, please provide the author with appropriate instructions. Also see ["When to rebase and squash commits"](https://docs.astropy.org/en/latest/development/when_to_rebase.html).
- [ ] Did the CI pass? If no, are the failures related? If you need to run daily and weekly cron jobs as part of the PR, please apply the `Extra CI` label. Codestyle issues can be fixed by the [bot](https://docs.astropy.org/en/latest/development/workflow/development_workflow.html#pre-commit).
- [ ] Is a change log needed? If yes, did the change log check pass? If no, add the `no-changelog-entry-needed` label. If this is a manual backport, use the `skip-changelog-checks` label unless special changelog handling is necessary.
- [ ] Is this a big PR that makes a "What's new?" entry worthwhile and if so, is (1) a "what's new" entry included in this PR and (2) the "whatsnew-needed" label applied?
- [ ] Is a milestone set? Milestone must be set but `astropy-bot` check might be missing; do not let the green checkmark fool you.
- [ ] At the time of adding the milestone, if the milestone set requires a backport to release branch(es), apply the appropriate `backport-X.Y.x` label(s) *before* merge.

### CODE REVIEW

**Review by nstarman — APPROVED** [review]

**Comment by nstarman** [comment]
Thanks @WilliamJamieson! The pytest command specification got a bit verbose, but short of a refactoring, I think it's good.

**Comment by WilliamJamieson** [comment]
@nstarman if you think this PR is ready for string normalization, please move it to the "Ready for string normalization" column in https://github.com/orgs/astropy/projects/10.

**Comment by WilliamJamieson** [comment]
> @nstarman if you think this PR is ready for string normalization, please move it to the "Ready for string normalization" column in [github.com/orgs/astropy/projects/10](https://github.com/orgs/astropy/projects/10).

Thanks for marking this for string normalization. I have normalized strings. If the results look good please move this the the "Ready for Squash" column.

File: astropy/tests/runner.py [hunk 1]
```diff
@@ -156,8 +158,7 @@ def _generate_args(self, **kwargs):
 
         return args
 
-    RUN_TESTS_DOCSTRING = \
-        """
+    RUN_TESTS_DOCSTRING = """
```
Comment (nstarman) [inline astropy/tests/runner.py:161]: Is this an API change to the docs?

File: astropy/tests/runner.py [hunk 2]
```diff
@@ -558,17 +576,22 @@ def docs_path(self, docs_path, kwargs):
...
+                    "Can not test .rst docs for {name}, since "
+                    "docs path ({path}) does not exist."
+                )
+                paths = self.packages_path(
+                    kwargs["package"], docs_path, warning=warning_message
+                )
+            elif not kwargs["test_path"]:
+                paths = [
+                    docs_path,
+                ]
```
Comment (nstarman) [inline astropy/tests/runner.py:591]:
```suggestion
                paths = [docs_path]
```

**Review by nstarman — COMMENTED** [review]
Going over with a finer-tooth comb I spotted two small things.

File: astropy/tests/runner.py (hunk 1 above)
Comment (WilliamJamieson) [reply astropy/tests/runner.py:161]: I don't think this will be an API change since thats just fixing the line return.

**Review by astrofrog — APPROVED** [review]
Looks good to me!

**Pull request merged**